src/kunden.js

```diff
@@ -358,7 +372,7 @@ exports.setVisited = function (id, log, SEP) {
       else {
         value = value.replace(/"/g, '""');
         value = value.replace(/\n/g, '\\n');
```

Comment:
Ahh das ist son Windows Mist oder? Muss ich vermtulich in meiner Funktion auch noch anpassen